src/storage/resourceLoader.ts

```diff
@@ -211,33 +215,30 @@ export function correlateTopicsWithSchemas(
  * Deep read and return of all schemas in a CCloud or local environment's Schema Registry.
  * Does not store into the resource manager.
  *
- * @param schemaRegistryId The Schema Registry ID to fetch schemas from (within the environment).
- * @param connectionId The connection ID to use to fetch schemas.
- * @param environmentId Optional: the CCloud environment ID to associate CCloud schemas with.
+ * @param schemaRegistry The Schema Registry to fetch schemas from.
  * @returns An array of all the schemas in the environment's Schema Registry.
  */
-export async function fetchSchemas(
-  schemaRegistryId: string,
-  connectionId: string,
-  environmentId: string | undefined = undefined,
-): Promise<Schema[]> {
+export async function fetchSchemas(schemaRegistry: SchemaRegistry): Promise<Schema[]> {
```

Comment:
Obligatory: @jlrobins was right! (We just weren't ready for local env support until now 😎)

Reply:
Eventual consistency!